src/generateNumber.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const { MAX_NUMBER, MIN_NUMBER } = require('./constants');
+
+function generateNumber() {
+  return Math.round(Math.random() * (MAX_NUMBER - MIN_NUMBER) + MIN_NUMBER);
+}
```

Comment:
<img width="500" alt="image" src="https://github.com/mate-academy/node_bulls-and-cows/assets/86789161/35d1a183-394e-4b28-8e78-2dc3fd2e0ca7">

Your function doesn't generate number with different digits
<img width="482" alt="image" src="https://github.com/mate-academy/node_bulls-and-cows/assets/86789161/67e87478-cb5b-4c07-b47d-2466e27c4ad1">